config/config-api/src/test/java/com/thoughtworks/go/config/materials/AbstractMaterialConfigTest.java

```diff
@@ -15,87 +15,85 @@
  */
 package com.thoughtworks.go.config.materials;
 
-import com.thoughtworks.go.config.*;
+import com.thoughtworks.go.config.CaseInsensitiveString;
+import com.thoughtworks.go.config.PipelineConfig;
+import com.thoughtworks.go.config.PipelineConfigSaveValidationContext;
+import com.thoughtworks.go.config.ValidationContext;
 import com.thoughtworks.go.config.materials.dependency.DependencyMaterialConfig;
 import com.thoughtworks.go.config.materials.mercurial.HgMaterialConfig;
 import com.thoughtworks.go.domain.BaseCollection;
 import com.thoughtworks.go.domain.materials.MaterialConfig;
-import org.hamcrest.Matchers;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.util.HashMap;
 import java.util.Map;
 
-import static org.hamcrest.Matchers.not;
-import static org.hamcrest.Matchers.sameInstance;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.nullValue;
-import static org.junit.Assert.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.*;
 
-public class AbstractMaterialConfigTest {
+class AbstractMaterialConfigTest {
     @Test
-    public void shouldRecomputePipelineUniqueFingerprint_whenAttributesChanged() {
+    void shouldRecomputePipelineUniqueFingerprint_whenAttributesChanged() {
         TestMaterialConfig testMaterialConfig = new TestMaterialConfig("foo");
 
         String pipelineUniqueFingerprint = testMaterialConfig.getPipelineUniqueFingerprint();
         testMaterialConfig.setConfigAttributes(m("bar", "baz"));
-        assertThat(testMaterialConfig.getPipelineUniqueFingerprint(), not(pipelineUniqueFingerprint));
+        assertThat(testMaterialConfig.getPipelineUniqueFingerprint()).isNotEqualTo(pipelineUniqueFingerprint);
     }
 
     @Test
-    public void shouldNotSetMaterialNameIfItIsSetToEmptyAsItsAnOptionalField() {
+    void shouldNotSetMaterialNameIfItIsSetToEmptyAsItsAnOptionalField() {
         AbstractMaterialConfig materialConfig = new TestMaterialConfig("");
         Map<String, String> map = new HashMap<>();
         map.put(AbstractMaterialConfig.MATERIAL_NAME, "");
 
         materialConfig.setConfigAttributes(map);
 
-        assertThat(materialConfig.getName(), is(nullValue()));
+        assertThat(materialConfig.getName()).isNull();
     }
 
     @Test
-    public void shouldRecomputeSqlCriteriaAndXmlAttributeMap_whenAttributesChanged() {
+    void shouldRecomputeSqlCriteriaAndXmlAttributeMap_whenAttributesChanged() {
         AbstractMaterialConfig testMaterialConfig = new TestMaterialConfig("foo");
 
         Map<String, Object> sqlCriteria = testMaterialConfig.getSqlCriteria();
         testMaterialConfig.setConfigAttributes(m("bar", "baz"));
-        assertThat(testMaterialConfig.getSqlCriteria(), not(sameInstance(sqlCriteria)));
-        assertThat(testMaterialConfig.getSqlCriteria().get("foo"), is("baz"));
+        assertThat(testMaterialConfig.getSqlCriteria()).isNotEqualTo(sqlCriteria);
```

Comment:
@GaneshSPatil  I hope the conversion was correct.

Reply:
@kritika-singh3 - https://github.com/gocd/gocd/pull/8013/files#diff-1ff479f423bf1a1145900a784673600aR52-R53 is the only change which can possibly cause the spec failures. I am not sure why this would fail.

Also, we must bring back `assertThat(testMaterialConfig.getSqlCriteria().get("foo"), is("baz"));` assertion.

Reply:
This is just the conversion from Junit4 assertion to assertj assertions.
That line is still present.

Reply:
Ahh Okay. Got it. Thanks.